### DISCUSSION TITLE
I forget my password.

### DISCUSSION THREAD

**Discussion by LittleNewton** [zbm-dev/zfsbootmenu#688] [Questions] [answered]
Is there a way to enter the boot selection menu of Debian 12?

**Comment by LittleNewton** [comment]
Once I select the zbm boot entry, it would directly get into the plasma KDE lock screen. I cannot enter recovery mode.

**Comment by ahesford** [comment, marked as answer]
You can use `Ctrl+E` to edit the kernel command-line for your boot environment to allow you to boot to a recovery mode, or you can use `Ctrl+J` to chroot into your boot environment and just reset the password there.

**Comment by LittleNewton** [comment]
Unfortunately, I can not set my password. The experience with Debian 12 Plasma KDE is unpleasant.

- Debian 12 will suspend the initial session and I cannot set the later things anymore; `[fixed]`
- I have set the password for root but I cannot log into it in the KDE desktop environment, or SSH; `[need-to-do]`

If I find out how to fix it, I can help with the enhancement of the documents, and make it easier to use.

**Reply by LittleNewton** [reply]
- Press `Ctrl E` to edit the boot command, add `init=/bin/bash` behind `quiet`; then boot the OS.
- Add a new user using `sudo useradd -u <USER_ID> -m -s /bin/bash ${USERNAME}`; 
- Change password of root using `passwd`;
- Log into the new account using ssh and do what you want.
